app/assets/stylesheets/application/components/ann-work-profile-light.scss

```diff
@@ -76,5 +76,29 @@
         }
       }
     }
+
+    .prev-episode, .next-episode {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line